crates/librqbit/README.md

```diff
@@ -1 +1,128 @@
+                for (idx, torrent) in torrents.iter().enumerate() {
+                    match &torrent.state {
+                        ManagedTorrentState::Initializing => {
+                            println!("[{}] initializing", idx);
+                        },
+                        ManagedTorrentState::Running(handle) => {
+                            let peer_stats = handle.torrent_state().peer_stats_snapshot();
+                            let stats = handle.torrent_state().stats_snapshot();
+                            let speed = handle.speed_estimator();
+                            let total = stats.total_bytes;
+                            let progress = stats.total_bytes - stats.remaining_bytes;
+                            let downloaded_pct = if stats.remaining_bytes == 0 {
+                                100f64
+                            } else {
+                                (progress as f64 / total as f64) * 100f64
+                            };
+                            println!(
+                                "[{}]: {:.2}% ({:.2}), down speed {:.2} MiB/s, fetched {}, remaining {:.2} of {:.2}, uploaded {:.2}, peers: {{live: {}, connecting: {}, queued: {}, seen: {}}}",
+                                idx,
+                                downloaded_pct,
+                                SF::new(progress),
+                                speed.download_mbps(),
+                                SF::new(stats.fetched_bytes),
+                                SF::new(stats.remaining_bytes),
+                                SF::new(total),
+                                SF::new(stats.uploaded_bytes),
+                                peer_stats.live,
+                                peer_stats.connecting,
+                                peer_stats.queued,
+                                peer_stats.seen,
+                            );
+                        },
+                    }
+                }
+            });
+            tokio::time::sleep(Duration::from_secs(1)).await;
+        }
+    };
+
+    // Create the torrent session, one session can have more than one torrent at the same time
+    let session = Arc::new(
+        Session::new(PathBuf::from("C:\\Anime"), spawner).await?
+    );
+
+    // Spawn the properties printer function
+    spawn(stats_printer(session.clone()));
+
+    // Add the magnet link to the torrent session, you don't have to specify if it's an url or magnet,
+    // the library will recognize automatically for you
+    let handle = match session.add_torrent(MAGNET_LINK, None).await {
+        Ok(v) => match v {
+            AddTorrentResponse::AlreadyManaged(handle) => {
+                println!(
+                    "torrent {:?} is already managed, downloaded to {:?}",
+                    handle.info_hash, handle.output_folder
+                );
+                Err(())
+            }
+            AddTorrentResponse::ListOnly(ListOnlyResponse {

```

Comment:
there's no way to pass "listOnly" in the example code, so this branch can be removed